README.md

```diff
@@ -833,6 +833,12 @@ when you need to retrieve a single record by some attributes.
   Mitigate code duplication by using partial templates and layouts.
 <sup>[[link](#partials)]</sup>
 
+* <a name="no-instance-variables-in-partials"></a>
+  Never use instance variables in partials, pass a local variable to `render` instead. Partial may be used
```

Comment:
I'd change `never` to `avoid`, as plenty of people are doing so. Some concrete examples here would be helpful.